backend/src/main/resources/liquibase/db-changelog-2.0.xml

```diff
@@ -325,4 +325,22 @@
         <tagDatabase tag="db_version_2.15"/>
     </changeSet>
 
+    <changeSet id="add-created-on-project" author="cory">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="project" columnName="created"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="project">
+            <column name="created" type="boolean"/>
```

Comment:
Is there a way we can default it to 1 in the liquibase change? This will allow all previously existing ones to work correctly.

Reply:
:+1: